src/profile_manager/templates/profile_manager/snippets/staff_list.html

```diff
@@ -0,0 +1,121 @@
+            Avatar
+        </th>
+        <th data-sortable='true' data-field='first' data-visible="false">
+            First Name
+        </th>
+        <th data-sortable='true' data-field='preferred'>
+            Preferred Name
+        </th>
+        <th data-sortable='true' data-field='last'>
+            Last Name
+        </th>
+        <th data-sortable='true' data-field='alias'
+            {% if request.user.is_staff %}data-visible="false"{% endif %}>
+            Alias
+        </th>
+        <th data-sortable='true' data-field='portfolio'
+            {% if request.user.is_staff %}data-visible="false"{% endif %}>
+            Portfolio
+        </th>
+
+        {% if request.user.is_staff %}
+
+        <!-- Uses `last_login_datetime` for sorting-->
+        <th data-sortable='true' data-field='last_login' data-sort-name='last_login_datetime' data-visible="false">
+            Last Login
+        </th>
+        <th data-field='last_login_datetime' data-switchable='false' data-visible='false'></th>
+        <!-- End sorting group -->
+
+        <th data-sortable='true' data-field='username'>
+            Username
+        </th>
+        <th data-field='skipped' data-switchable="false">
+            Admin
+        </th>
+        {% endif %}
+
+        </tr>
+    </thead>
+
+    <tbody>
+        {% for object in object_list %}
+
+        <tr {% if request.user.is_staff %}
+                {% if object.not_earning_xp or object.banned_from_comments %}
+                    class="warning"
+                {% elif object.gone_stale %}
+                    class="danger"
+                {% endif %}
+            {% endif %}>
+
+            <td><img class="icon-md img-rounded" src="{{ object.get_avatar_url }}" alt="avatar"/></td>
+            <td>{{ object.user.first_name }}</td>
+            <td>{{ object.get_preferred_name }}</td>
+            <td>{{ object.user.last_name|default:"-" }}</td>
+            <td>{{ object.alias_clipped|default:"-" }}</td>
+
+            <td>{% if object.user.portfolio.listed_locally or request.user.is_staff %}
+                    <a href="{{ object.user.portfolio.get_absolute_url }}">View</a>
+                {% else %}<i class="fa fa-eye-slash"></i>{% endif %}
+            </td>
+
+            {% if request.user.is_staff %}
+                <td>
+                    {{ object.user.last_login|date:'y/m/d' }}<br>
+                    <small>{{ object.user.last_login|timesince }} ago</small>
+                </td>
+                <!-- `last_login_datetime` -->
+                <td>
+                    {{ object.user.last_login|date:"y/m/d" }} {{ object.user.last_login|time:"H:i:s"}}
+                </td>
+                <!-- end `last_login_datetime` -->
+                <td><a href="{% url 'profiles:profile_detail' object.id %}">{{ object.user.username|truncatechars:14 }}</a></td>
+
+                {#      Admin buttons #}
```

Comment:
See comments below, but I think the admin buttons can be removed when viewing staff list